Failing Kiwix if started from Windows Network Drive (was: Kiwix does not start on Windows: "QT5Core.dll was not found")

I noticed that Kiwix (version: kiwix-desktop_windows_x64_2.3.0) can't start properly from mapped network shares.
I got "The code execution cannot proceed because QT5Core.dll was not found. Reinstalling the program may fix this problem."
On regular drive everything works fine.

This is because #1178 doesn't compile on Windows.
See error log at https://ci.appveyor.com/project/Kiwix/kiwix-build/builds/50423338

```
src\main.cpp(29): error C2664: 'int MessageBoxA(HWND,LPCSTR,LPCSTR,UINT)': cannot convert argument 2 from 'QString' to 'LPCSTR'
```

Why not using QMessageBox instead of windows low level api `MessageBoxA` ?

--- a/src/main.cpp
+++ b/src/main.cpp
@@ -26,7 +26,11 @@ int main(int argc, char *argv[])
     std::string driveLetter = kiwix::getExecutablePath().substr(0, 3);
     UINT driveType = GetDriveTypeA(driveLetter.c_str());
     if(driveType == DRIVE_REMOTE) {
-        int msgboxID = MessageBoxA(NULL, gt("disable-sandbox"), gt("about-kiwix-desktop-title"), MB_YESNO | MB_ICONQUESTION);
+        const std::wstring messageStr = gt("disable-sandbox").toStdWString();
+        const std::wstring titleStr = gt("about-kiwix-desktop-title").toStdWString();
+        const wchar_t* message = message_str.c_str();
+        const wchar_t* title = title_str.c_str();
+        int msgboxID = MessageBoxExW(NULL, message, title, MB_YESNO | MB_ICONQUESTION, 0);
         if (msgboxID == IDYES)
             qputenv("QTWEBENGINE_CHROMIUM_FLAGS", QByteArray("--no-sandbox"));
     }

